### PR TITLE
First function

### PR DESCRIPTION
**Pull request by tomkr**
Add a function that gets the first result.

### CODE REVIEW

File: lib/poms/api/request.rb [hunk 1]
```diff
@@ -39,10 +39,10 @@ class PostRequest
       include RequestExecution
       attr_reader :uri, :credentials, :body
 
-      def initialize(uri, credentials, body: {})
+      def initialize(uri, credentials, body = {})
```
Comment (mxgrn) [inline lib/poms/api/request.rb]: Why do we need another PR that does exactly the same thing?

File: lib/poms.rb [hunk 2]
```diff
@@ -19,6 +19,13 @@ def configure
     nil
   end
 
+  # Returns the first result for a mid if it is not an error.
```
Comment (mxgrn) [inline lib/poms.rb]: I don't get the meaning of the "if it is not an error" amendment. For example, I don't see any check for errors.

File: lib/poms/api/request.rb (hunk 1 above)
Comment (tomkr) [inline lib/poms/api/request.rb]: Because this shouldn't have been part of this PR, it's the same commit which snuck into this PR.

File: lib/poms.rb (hunk 2 above)
Comment (tomkr) [inline lib/poms.rb]: `unless item.key?("error")`?

Comment (mxgrn) [inline lib/poms.rb]: I was blind, sorry.
